examples/react-native/storybook/stories/PhoneNumberField.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { StyleSheet } from 'react-native';
+import { storiesOf } from '@storybook/react-native';
+
+import {
+  PhoneNumberField,
+  PhoneNumberFieldProps,
+} from '@aws-amplify/ui-react-native/dist/primitives';
+
+const styles = StyleSheet.create({
+  container: {
+    width: '75%',
+  },
+  inputStyle: {
+    width: '50%',
+  },
+  picker: {
+    color: 'red',
+    fontSize: 16,
+    width: '50%',
+  },
+  pickerItem: {
+    color: 'red',
+    fontSize: 16,
+  },
+});
+
+const codes = ['+1', '+7', '+20', '+27', '+30'];
+const props: PhoneNumberFieldProps = {
+  style: styles.container,
+  defaultDialCode: '+1',
```

Comment:
Can we have a separate story for `defaultDialCode`?

Reply:
Not sure I follow, what would this story showcase?